quicktile/commands.py

```diff
@@ -183,6 +183,20 @@ def call(self, command, winman, *args, **kwargs):
             logging.error("Unrecognized command: %s", command)
             return False
 
+    def call(self, command, winman, *args, **kwargs):
```

Comment:
There's no expectation that a completely unadorned name like `call` on a command registry should support a list of calls.

Please rename this to something like `call_multiple`. The places you'll need to adjust to support the new name are:

* At the end of `main` in `__main__.py`, where command-line input is handled
* At the end of `doCommand` in `dbus_api.py`
* In the `call` closure at the end of `keybinder.py`